services/business/checkers/route_rules/route_checker.go

```diff
@@ -11,6 +11,13 @@ import (
 
 type RouteChecker struct{ kubernetes.IstioObject }
 
+// RouteChecker checks if the current RouteRule has:
+// 1. All weights with a numeric number.
+// 2. All weights have value between 0 and 100.
+// 3. Sum of all weights are 100 (if only one weight, then it assumes that is 100).
+// 4. All the route has to have weight label.
+// Output: It builds a IstioCheck whenever has an invalid value for precedence.
```

Comment:
Same comment here with the Output, I would try to follow same style used in other areas.